pypeit/core/telluric.py

```diff
@@ -0,0 +1,1223 @@
+
+
+
+import numpy as np
+import scipy
+import matplotlib.pyplot as plt
+import os
+from sklearn import mixture
+from astropy.io import fits
+from pypeit.core import pydl
+import astropy.units as u
+from astropy.io import fits
+from astropy import stats
+from pypeit.core import load, flux_calib
+from pypeit.core.wavecal import wvutils
+from astropy import table
+from pypeit.core import save
+from pypeit.core import coadd2d
+from pypeit.core import coadd1d
+from pypeit.spectrographs import util
+from pypeit import utils
+from pypeit import msgs
+import pickle
+PYPEIT_FLUX_SCALE = 1e-17
```

Comment:
This is declared in multiple files.  I'll first give my usual complaint that we shouldn't be declaring globals.  Second, we run the risk of inconsistency by having to remember to change this in multiple places.  Maybe we should have a `constants` module where all of this is defined?  Third, it's helpful if you organize imports, starting from things from the python standard library (like os and sys), then moving steadily toward more specialized packages and finally ending with local modules.  This will help you not execute `from astropy.io import fits` three times...

Reply:
imports rearranged. Global variable removed.